docs/application/vscode-ext/web.md

```diff
@@ -30,11 +30,11 @@ To install VS Code Extension for Tizen Web from the Visual Studio Code Marketpla
 
   - To perform a fresh installation, click **YES**.
 
-    To perform a fresh installation and to begin the Tizen Baseline SDK installation, click YES. Else, click No, if you have already done the installation.
+    To perform a fresh installation and to begin the Tizen Baseline SDK installation, click **YES**. Else, click **NO**, if you have already done the installation.
```

Comment:
Question:. I can't confirm this step. When does the Reload button appear? I tried to install it and it was installed after step 3. Please be clear.
 See https://code.visualstudio.com/updates/v1_31#_no-reload-on-install
=> Reload depends on the environment compared to vscode.If the Reload button appears, click to open the vscode.
However, if the button does not appear, the user simply shuts down the vscode and runs it again.

Question:. I can't confirm the second part of this step. "Are we referring to the Tizen Baseline SDK for Intelligence for Tizen Application? If so, it is recommended to mention the SDK name in detail. Secondly, as far as I know, it is installed automatically. Ask for more information.
=> Tizen Baseline SDK name is Unique. We can't change it. Therefore, keep the existing name.

Global description: If only "Open Command Palette" is written, it is not clear, nor is it clear how and where to open. Therefore, it is better to mention the steps on how to open it.
=> The vscode user knows how to run "Open Command Palette" by default.


I've modified all the remaining parts that need to be fixed.

done.

Reply:
> > 1. In the Tizen Web search result that appears, click Install.
> 
> Isn't it better to let user know that we will choose "Visual Studio Code Extension for Tizen Web" among the search result of "Tizen Web"?

@mijin85cho, I tested with the application and we cannot search using "Visual Studio Code Extension for Tizen Web", we can only search using "Tizen Web". 

Please correct me in case i have missed something.

Reply:
@surbhig12 I've heard that "Visual Studio Code Extension for Tizen Web" is not published yet, waiting the review & merge of this page. 
When this PR is merged and published on the Developer site, Extension itself will also be published.

@DohyungLim Please check my answer.

Reply:
@mijin85cho, I tested with the application and we cannot search using "Visual Studio Code Extension for Tizen Web", we can only search using "Tizen Web".

Please correct me in case I have missed something.

Reply:
Check out "Install from Marketplace".
It is correct to search for "Tizen Web".
"Visual Studio Code Extension for Tizen Web" is the name to be posted to the site.

done.